From 54d2912c19240a1a1866ba1519889086868038c4 Mon Sep 17 00:00:00 2001
From: Guy Harris <guy@alum.mit.edu>
Date: Tue, 11 Nov 2014 19:18:12 -0800
Subject: [PATCH 3/4] Not using offsetof() any more, so no need for <stddef.h>.

---
 print-aodv.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/print-aodv.c b/print-aodv.c
index 2649936..9b3523a 100644
--- a/print-aodv.c
+++ b/print-aodv.c
@@ -37,9 +37,6 @@
 
 #include <tcpdump-stdinc.h>
 
-/* for offsetof */
-#include <stddef.h>
-
 #include "interface.h"
 #include "addrtoname.h"
 #include "extract.h"			/* must come after interface.h */
-- 
1.8.3.1

